pkg/anypoint/client.go

```diff
@@ -7,15 +7,15 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"net/http"
 	"time"
 
-	"github.com/Axway/agent-sdk/pkg/cache"
+	"github.com/sirupsen/logrus"
```

Comment:
Can you please use "github.com/Axway/agent-sdk/pkg/util/log" instead to take advantage of global log config?

Reply:
Hey Vivek, we like to use logrus directly because it lets us log with extra fields, making the logs easier to read. The sdk package doesn't provide logging with fields, so it isn't as useful. Maybe the sdk can be enhanced to support this?